### DISCUSSION TITLE
Feedback and bugs from Capital Planning team

### DISCUSSION THREAD

**Discussion by TylerMatteo** [NYCPlanning/ae-cp-map#115] [General]
Hi @mbh329 @sallywuhoo! Let's use this discussion to capture feedback and bugs. You can collect everything in one comment below or make separate comments for each item, which ever is easier for you.

**Comment by sallywuhoo** [comment]
Hi Tyler - thanks for setting this discussion up!

Below is some of our initial feedback on the website:
1) After fully zooming out using the mouse scroller, any more scrolls move the map to the left/down
2) The points representing each capital project are small and difficult to see on borough level (need to zoom into block level to see)
3) When the "District Type" filter is on, at certain zoom levels, some district borders disappear
![zoom borders](https://github.com/user-attachments/assets/a8f0b64e-b003-45e2-a131-b154f2115b8e)
4) Capital projects that are "citywide" zoom far out when clicked, it's difficult to see where the original point was located and what other points are also highlighted (e.g. click on "Citywide Seawall Reconstruction" located in Astoria)
![image](https://github.com/user-attachments/assets/458263dc-4815-451d-bae6-fba116f53fbe)
5) When a large area (i.e. park) is selected, it's not possible to select a point within that area because they become grayed out
![image](https://github.com/user-attachments/assets/e146fce2-b33f-4e83-ba1c-bcb0337fafba)
6) After the filters are used (district type, borough, district number) and the "Go to Selected District" button is pressed, trying to go back to an unselected map is difficult without resetting the URL (because the selected district is still highlighted even after changing "District Type" back to "-Select-")
![image](https://github.com/user-attachments/assets/653c30a6-12d6-4788-af4f-43abdbed0520)

Let me know if you have any questions, and I'm happy to go through them in person/screen share!

**Reply by TylerMatteo** [reply]
Thanks @sallywuhoo! @mbh329 It looks like 1 and 3 are bugs and the rest are feedback that @jessicashanshanhuang can factor into future iteration. Do you see any of these as blockers for the initial release?

**Reply by sallywuhoo** [reply]
Hi @TylerMatteo - just spoke to Max and nope, we're not seeing any of them as blockers for the initial release. Thanks!

**Comment by TylerMatteo** [comment]
@sallywuhoo @mbh329 We have the messaging for CPE ready to go up! You can preview it here - [https://staging--cp-platform.netlify.app/capitalprojects](https://staging--cp-platform.netlify.app/capitalprojects). If it looks good let us know and we'll put it live.

**Reply by sallywuhoo** [reply]
@TylerMatteo looks good, thank you!

**Comment by horatiorosa** [comment]
Note: the messaging will show up once on the [capitalprojects page](https://staging--cp-platform.netlify.app/capitalprojects)  but will always appear on [the table page](https://staging--cp-platform.netlify.app/capitalprojects/table) and on the [map page](https://staging--cp-platform.netlify.app/map/capitalprojects#10/40.7128/-74.0807)

**Comment by sallywuhoo** [comment]
Adding some other notes we're seeing on the app for future iterations:

7. Would be helpful to have a feature that allows searches by address or project ID (or in general, more filter and search functions)

8. Extension to the previous point 5 - at some locations, there are multiple dots overlapped and unable to click the one underneath (e.g. 100 Centre Street Toilet Repairs)
![image](https://github.com/user-attachments/assets/4f7059e0-03a3-448d-a39f-3d5b11c973ba)
